### ISSUE TITLE
Import issue : You may need an additional loader to handle the result of these loaders.

### ISSUE THREAD

**Issue by jasonakon** [edisonneza/jspdf-invoice-template#5]
![image](https://user-images.githubusercontent.com/13328627/122595150-99466300-d09a-11eb-8053-61e60350cd0c.png)

This error message happens right after imported your package

**Comment by edisonneza** [comment]
Are you trying to import it in a nodejs file, or in a frontend js file?

**Comment by jasonakon** [comment]
Hi, in frontend js file

**Comment by edisonneza** [comment]
I had a similar issue about these but I thought it was fixed.  Probably is caused by the npm version because in my tests I did not encounter it.  Anyway I will remove the optional chaining package asap.

**Comment by vegesriharsha** [comment]
Thanks for sharing the template. I am encountering the same issue that @jasonakon reported after I imported the package.

**Comment by bhasshaw** [comment]
Any update on this issue? When will you be fixing this?

**Comment by edisonneza** [comment]
Yeah, I'll do my best. I think this weekend I'll publish new version.

**Comment by slimlime** [comment]
Adding stdout text copy paste here for searchability

```sh
Error: ./node_modules/jspdf-invoice-template/src/index.js 90:22
Module parse failed: Unexpected token (90:22)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|     orientationLandscape: props.orientationLandscape || false,
|     logo: {
>       src: props.logo?.src || "",
|       width: props.logo?.width || "",
|       height: props.logo?.height || "",
```



Is updating the ECMAScript target sufficient to resolve the modern syntax? 


---

i.e. was the loader issue caused by https://github.com/edisonneza/jspdf-invoice-template/commit/3751f8b7a72727e77cbd2a1c736cb82a5923cbd3#diff-b55cdbef4907b7045f32cc5360d48d262cca5f94062e353089f189f4460039e0


---
The babel plugin would have fixed this?

https://github.com/edisonneza/jspdf-invoice-template/commit/78e783853247c8678430e7d3d20cfdd424339620#diff-09c56b2bf95de2a608a36afef3b6893146a959d6739be0a154dc9c9f02d80f24


This is related to https://github.com/edisonneza/jspdf-invoice-template/issues/2 #2

**Comment by slimlime** [comment]
I used a quick online transpiler tool to fill the modern syntax into compatible js for those optional chaining accessors etc  `./node_modules/jspdf-invoice-template/src/index.js 90:22` to start prototyping with it faster (unreadable code, though).

TypeScript <3

Thank you for the package.
Looking forward to seeing how you update this.

**Comment by edisonneza** [comment]
> I used a quick online transpiler tool to fill the modern syntax into compatible js for those optional chaining accessors etc  `./node_modules/jspdf-invoice-template/src/index.js 90:22` to start prototyping with it faster (unreadable code, though).
> 
> TypeScript <3
> 
> Thank you for the package.
> Looking forward to seeing how you update this.
> 

Thanks man! I'll do my best. I have been too busy and couldn't do the updates till now. I think and hope these days I'll push them.

**Comment by edisonneza** [comment]
Please, can someone let me know which version of nodejs and npm have you installed? 
Cmd commands: 
npm -v and node -v

**Comment by bhasshaw** [comment]
npm version 6.14.13 and node version 14.17.2

**Comment by edisonneza** [comment]
@bhasshaw please, can you try to update npm and then try if this issue is still present? (I'm trying to understand the issue so I can leave the optional chaining package). Thank you!

**Comment by bhasshaw** [comment]
I'm using a .nvmrc file with the node version 14.17.2, that then determines the npm version of 6.14.13 so there is no updating it beyond that.

**Comment by bhasshaw** [comment]
I updated node to 16.5.0 and nvm updated npm to 7.19.1 and now it works.

**Comment by edisonneza** [comment]
I'm gonna close this issue due to the fix done in version 1.3.2

**Comment by edisonneza** [comment]
Please, if any of you can test the package in your environment that you had this issue, can you let me know if the issue persists or not? Thanks!
